standards/aws-tags.md

```diff
@@ -0,0 +1,35 @@
+# Using AWS Tags
+
+[Amazon's documentation](https://aws.amazon.com/answers/account-management/aws-tagging-strategies/) describes tags as:
+
+ >...a simple label consisting of a customer-defined key and an optional value that can make it easier to manage, search for, and filter resources. Although there are no inherent types of tags, they enable customers to categorize resources by purpose, owner, environment, or other criteria.
+
+Using tags helps us identify stakeholders as well as track critical financial metrics
+about our AWS usage. You MUST add Amazon tags to services when they are available.
+
+## Required Tags
+
+| Key           | Description                                                                 |
+|:--------------|:----------------------------------------------------------------------------|
+| Project       | The primary project this resource is being used for, or "UnknownProject".   |
+| OtherProjects | Any other projects that rely on the resource being up ('/'-separated list). |
+| Environment   | `production`, `qa`, `UnknownEnvironment`, `development`                     |
+
+
+### Project Tags In Use
+
+Every effort SHOULD be made to maintain uniform capitalization when creating new
+tags using existing values;
+
+* DAM
+* DXP
+* LSP
+* NA
+* Shared
+* SimplyE
+
```

Comment:
Column A in that spreadsheet are the project tags.  Column B is the business domain -- a bunch of project will fall into each of the domains.  So then we can have cost calculations by project, and also aggregated into the four units within the department.

Reply:
Gotcha - thanks for clarifying that.

What do you think is better...

1.  We make this repo the cannonical listing of Project tag values.
2.  I switch this documentation to link to your spreadsheet?